netlify.toml

```diff
@@ -1,23 +1,17 @@
 [build.environment]
    NODE_VERSION = "16"
-   YARN_VERSION = "3.2.3"
+   YARN_VERSION = "3.2.4"
 
 [[plugins]]
-    # Netlify does not support build plugins installed with Yarn v2/3 PnP,
-    # so we need to use a local version for now.
-    # The unused package is still included in package.json
-    # to better surface updated releases.
-    #
-    # See:
-    # - https://answers.netlify.com/t/build-plugin-error-netlify-plugin-x-plugin-not-installed-or-found/7625/4?u=jgerigmeyer
-    # - https://github.com/netlify/build-image/issues/612
-    # - https://github.com/netlify/build/issues/1535
-    package = "/netlify-plugins/netlify-plugin-11ty"
+    package = "netlify-plugin-11ty"
 
     [plugins.inputs]
         cache_img = "./assets/images"
         cache_assets = false
 
+[[plugins]]
+    package = "@netlify/plugin-a11y"
```

Comment:
@mirisuzanne @oluoluoxenfree This allows us to now configure the plugin, if we decide we don't want the default settings: https://github.com/netlify-labs/netlify-plugin-a11y#configuration

Reply:
Great! Since we always review changes in a PR with a deploy preview, I think we should turn off `failWithIssues`. It's useful for us to have deploy previews that build, even when there are still changes needed.